torch/csrc/jit/codegen/cuda/ir_internal_nodes.h

```diff
@@ -258,78 +262,215 @@ class TORCH_CUDA_CU_API GroupedReductionOp : public Expr {
   bool is_allreduce_ = false;
 };
 
+//! Average, variance and N (count) vals for Welford
```

Comment:
Tried to clean the code up a little with a class to represent the three parameters of welford.